catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/charts/ChartResource.java

```diff
@@ -111,7 +111,7 @@ public ChartList(List<Chart> data, String beforeCursor, String afterCursor, int
   }
 
   static final String FIELDS = "owner,followers,tags";
-  public static final List<String> FIELD_LIST = Arrays.asList(FIELDS.replaceAll(" ", "").split(","));
+  public static final List<String> FIELD_LIST = Arrays.asList(FIELDS.replaceAll("\\s", "").split(","));
 
```

Comment:
In this case just replace is sufficient per Sonar Cloud guidelines.

Reply:
Let me know if I should change the replaceAll to replace, I can do that it'd be a minor change :)

Reply:
That works as well. You can also create a separate issue to do that.